util/cardano-sl-util.cabal

```diff
@@ -132,6 +132,7 @@ test-suite test
                        Test.Pos.Util.Golden
                        Test.Pos.Util.Modifier
                        Test.Pos.Util.ModifierSpec
+                       Test.Pos.Util.Parallelize
```

Comment:
Should this be `Test.Pos.Util.Parallel.Parallelize`?

Reply:
Yep it should be! Forgot to change this after the whole `develop` vs `develop-new` thing.